### PR TITLE
Grey Budget App v1.0

### PR DESCRIPTION
**Pull request by DebyGrey**
In his PR, I have 

- Completed the budget app project, meeting all requirements, please find the resources below:
 - Link to [Project demo](https://www.loom.com/share/7d652e6da7a74efaa734417cea3c0bc5?sid=25f5f2ef-834d-4842-aa1e-3e32ce2913d0)
 - Link to [Live demo](https://budget-app-eo2t.onrender.com/)

### CODE REVIEW

File: app/views/categories/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      <% @expenses.each do |expense| %>
+        <div class="card mb-3">
+          <div class="card-body d-flex justify-content-between">
+            <h5 class="card-title"><%= expense.name %></h5>
+            <p class="card-text">Amount: <%= expense.amount %></p>
+          </div>
+        </div>
+      <% end %>
+    </div>
+  </div>
```
Comment (Olamarx) [inline app/views/categories/show.html.erb:18]: - [x] You will notice that each transaction is not arranged based on the latest, the Eba is the last I added which is the latest and it is supposed to be up, I will encourage you to loop it the other way round
### Your project
image

### Project requirements
image

File: spec/features/splashscreen_index_spec.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    click_link 'Sign in'
+    expect(page).to have_current_path('/users/sign_in')
+  end
+
+  scenario 'Click sign up' do
+    visit root_path
+    click_link 'Sign up'
+    expect(page).to have_current_path('/users/sign_up')
+  end
+end
```
Comment (Olamarx) [inline spec/features/splashscreen_index_spec.rb]: - [x] I will encourage you to fix the tests that are failing as pointed out in the image below. 
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @DebyGrey,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Readme file professional ✔️
- Linters passed ✔️
- Gitflow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @DebyGrey, 
congrats 2

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**